src/dlgProfilePreferences.cpp

```diff
@@ -1626,6 +1606,19 @@ void dlgProfilePreferences::copyMap()
     mudlet::self()->setShowMapAuditErrors(savedOldAuditErrorsToConsoleEnabledSetting);
 }
 
+void dlgProfilePreferences::setLogDir()
+{
+    Host* pHost = mpHost;
+    if (!pHost) {
+        return;
+    }
+
+    QDir currentLogDir = QFileDialog::getExistingDirectory(this, tr("Select log directory"), logDirPath, QFileDialog::ShowDirsOnly);
```

Comment:
If the user presses `Cancel` then the return value from `QFileDialog::getExistingDirectory(`...`)` is an empty string - is ***that*** the intended means to clear the setting and revert to the default `<profileHomeDir>/logs` directory - if **not** then *you will need an additional "reset" button on this dialogue to preform that task {which is a nicer UI method than the user wiping a previous selection because they accidentally pressed the button to change the setting and then tried to back out by hitting the obvious "Cancel"* which might be expected to not make a change *setting}*!

Reply:
Good point! That wasn't intended and I hadn't even thought of that.

I'll add a reset button.